.gitignore

```diff
@@ -1,3 +1,4 @@
 .idea/
 .work/
 img/company/
+test.php
```

Comment:
@yanachkov-devix what would happen if you happen to have a `test.php` file in your Unit tests in the future?